src/VisualCoverage.Console/CmdDriver.cs

```diff
@@ -159,7 +172,10 @@ static void Main(string[] args)
                 CloverReport cloverreport = new CloverReport();
                 using (StreamWriter outfile = new StreamWriter(options.CloverOutput))
                 {
-                    outfile.Write(cloverreport.Execute(pe));
+                    // (C) Siemens AG, [2018] begin---> 
```

Comment:
Do we really need this line?

Reply:
Hi, I am ok with that. Should I remove it in my changes or will you do it?

Reply:
If you can, that would be better 😃